tensorflow/tools/ci_build/Dockerfile.android

```diff
@@ -5,7 +5,9 @@ MAINTAINER Jan Prach <jendap@google.com>
 # Copy and run the install scripts.
 COPY install/*.sh /install/
 RUN /install/install_bootstrap_deb_packages.sh
-RUN add-apt-repository -y ppa:openjdk-r/ppa
+RUN add-apt-repository -y ppa:openjdk-r/ppa \
```

Comment:
This should be three different run lines, or don't repeat the add-apt-repository -y. 

See tests.